rules/0400-openvpn_rules.xml

```diff
@@ -50,4 +50,28 @@
         <group>openvpn-error,gdpr_IV_35.7.d,gdpr_IV_32.2,</group>
     </rule>
 
+<!--
+ Author: Stuart Williams <stwilliams@workforcesoftware.com>
+    Date: 2019-01-29
+    Note: Enhanced decoder rules to handle LDAP integrated authentication
+Comments: Not sure how useful the first entry is but will 'log' it - could be useful in auditing numbers etc
+
+Jan 28 14:25:49 VPN-SERVER-05892 openvpn: LDAP bind failed: Invalid credentials (80090308: LdapErr: DSID-55555555, comment: AcceptSecurityContext error, data 775, v3839)
+Jan 28 14:25:49 VPN-SERVER-05892 openvpn: Incorrect password supplied for LDAP DN "CN=Harry T. Hacker,OU=business unit,OU=department,DC=domain,DC=com".
+-->
+
+  <rule id="81810" level="5">
```

Comment:
Can you change the rule ID to continue the numeric sequence of the OpenVPN rules? The last one is `81804` so this rule should be the `81805` and the other one the `81806`

Reply:
Updated as requested.